HHS/OPRE-OPS
# feat: ✨adds CAN Spending page BLI Table

- adds CAN Spending Page
- adds CAN Budget Line Table and friends

## What changed

- starts the CAN Spending page
- adds CAN Budget Line table

Does not include:

- Cards
- agreement data needed (TBD for now)

## Issue

#2672 #3030 #3031

## How to test

1. see e2e tests pass
2. goto cans list 
3. select fiscal year 2021
4. select CAN ending in 4426
5. goto CAN Spending Tab
6. see CAN Budget Line table

## Screenshots

<img width="972" alt="image" src="https://github.com/user-attachments/assets/f1d05e21-1b7a-404a-9a75-e5a8f80a7c45">

## Definition of Done Checklist
